@uportal/context-menu-navigation/src/components/ContextMenu.vue

```diff
@@ -14,26 +14,49 @@
                 v-for="(content, contentIndex) in column.content"
                 :key="`${columnIndex}-${contentIndex}`"
                 class="dropdown-submenu"
-                :class="[dropDirection, { show: selected === content.fname }]"
+                :class="[dropDirection, { show: selected === content.fname, maintenance: content.lifecycleState === 'MAINTENANCE' }]"
                 :ref="'content-' + content.fname"
             >
+                <span
+                    v-if="content.lifecycleState === 'MAINTENANCE'"
+                    class="dropdown-item dropdown-toggle"
+                    @mouseover="toggleContent(content)"
+                    @focus="toggleContent(content)"
+                    >{{ content.name }}</span
+                >
                 <a
+                    v-else
                     class="dropdown-item dropdown-toggle"
-                    :href="content.parameters.alternativeMaximizedLink"
+                    :href="content.alternativeMaximizedLink"
```

Comment:
why was the `parameters` part of the path removed here?

Reply:
Fixed